{{cookiecutter.repo_name}}/Makefile

```diff
@@ -67,7 +67,7 @@ compile_translations:
 fake_translations: extract_translations dummy_translations compile_translations
 
 pull_translations:
-	tx pull -a
```

Comment:
@amangano-edx no, please add the force flag. I explained why it's necessary in https://github.com/edx/course-discovery/pull/457.